### ISSUE TITLE
[Bug]: Incorrect tab order (too high `tabindex` in `linkAnnotation`)

### ISSUE THREAD

**Issue by dokugo** [mozilla/pdf.js#20050]
### Attach (recommended) or Link to PDF file

[sample.pdf](https://github.com/user-attachments/files/21010622/sample.pdf)

### Web browser and its version

Chrome 137

### Operating system and its version

Windows 11

### PDF.js version

5.3.31

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

No

### Steps to reproduce the problem

1. Clone this sample project: https://github.com/wojtekmaj/react-pdf/tree/main/sample/vite
2. Install dependencies, start the dev server
3. Open http://localhost:5173
4. Press Tab: the annotation element gets focused
5. Press Tab again: the file upload input is focused
6. Press Tab again: the link with "Nullam accumsan libero non eros" text is focused
4. Inspect the link in DevTools: its parent `section.linkAnnotation` has `tabindex="1000"`

This sample uses the `react-pdf` library, which is build upon `pdfjs-dist` 5.3.31, [see here](https://github.com/wojtekmaj/react-pdf/blob/main/packages/react-pdf/package.json#L45).

### What is the expected behavior?

Tab order should be predictable and use standard keyboard navigation with accessibility best practices. For the sample PDF, it should be:

1. First Tab: focus the file upload input
2. Second Tab: focus the annotation
3. Third Tab: focus the link

### What went wrong?

The current tab order is broken by `linkAnnotation` elements with `tabindex="1000"`. In order to have a proper navigation we should avoid using >0 tabindex values.

<details>
<summary>Screenshot</summary>

![Image](https://github.com/user-attachments/assets/70d29026-3d73-451a-85ce-2c7071f693e4)
</details>

**Comment by calixteman** [comment]
It's been fixed few days ago:
https://github.com/mozilla/pdf.js/pull/20013
